includes/functions.php

```diff
@@ -63,7 +63,12 @@ function get_stories( array $attrs = [], array $query_args = [] ) {
  * @return void
  */
 function render_theme_stories() {
-	$customizer = new Customizer();
+	$injector = Services::get_injector();
+	if ( ! method_exists( $injector, 'make' ) ) {
+		return;
+	}
```

Comment:
Are these `method_exists()` checks used for static analysis?

If so, you can actually adapt static analysis to work transparently. HEre's how I did that for PHPStan:
https://github.com/ampproject/amp-wp/tree/develop/tests/php/src/PhpStan

Also, you can enable auto-completion and proper analysis in PHPStorm via a PHPStorm meta file:
https://github.com/ampproject/amp-wp/blob/develop/.phpstorm.meta.php

Reply:
Also, keep in mind that performance-wise, it is preferable to use the container instead of the injector in cases like this, as the container is a simple hash lookup.

Reply:
Wow, this is cool. Great work. I think for now, let's just get this merged and I will follow up with copying this change as well.